### PR TITLE
bubble_sort.py: Add BubbleSort Algo

### PR DESCRIPTION
**Pull request by sameerchoubey**
This add bubble_sort.py which is the bubbleSort implementation in Python. The time complexity of this algorithm is O(n^2).

### CODE REVIEW

**Comment by sameerchoubey** [comment]
The Travis CI build is failing in Python. I'm not sure why this is happening. Once, the Travis build passes, I'll update the PR with the guidelines and let you know, so, that you can merge the PR.

File: bubble_sort/Python/bubble_sort.py [hunk 1]
```diff
@@ -0,0 +1,14 @@
+def bubbleSort(arr): #Function that performs the bubbleSort
+
+  n = len(arr) #Finding the length of the array using the len function
```
Comment (sangamcse) [inline bubble_sort/Python/bubble_sort.py]: Line contains following spacing inconsistencies:
- Trailing whitespaces.

*Origin: SpaceConsistencyBear, Section: `all.pyjava`.*

The issue can be fixed by applying the following patch:

```diff
--- a/tmp/tmpann5j0jr/bubble_sort/Python/bubble_sort.py
+++ b/tmp/tmpann5j0jr/bubble_sort/Python/bubble_sort.py
@@ -1,6 +1,6 @@
 def bubbleSort(arr): #Function that performs the bubbleSort
 
-  n = len(arr) #Finding the length of the array using the len function 
+  n = len(arr) #Finding the length of the array using the len function
 
   for i in range(n): # Traverse through all array elements 
     for j in range(0, n-i-1): # Traverse the array from 0 to n-i-1 
```

File: bubble_sort/Python/bubble_sort.py [hunk 2]
```diff
@@ -0,0 +1,14 @@
+def bubbleSort(arr): #Function that performs the bubbleSort
+
+  n = len(arr) #Finding the length of the array using the len function 
+
+  for i in range(n): # Traverse through all array elements
```
Comment (sangamcse) [inline bubble_sort/Python/bubble_sort.py]: Line contains following spacing inconsistencies:
- Trailing whitespaces.

*Origin: SpaceConsistencyBear, Section: `all.pyjava`.*

The issue can be fixed by applying the following patch:

```diff
--- a/tmp/tmpann5j0jr/bubble_sort/Python/bubble_sort.py
+++ b/tmp/tmpann5j0jr/bubble_sort/Python/bubble_sort.py
@@ -2,7 +2,7 @@
 
   n = len(arr) #Finding the length of the array using the len function 
 
-  for i in range(n): # Traverse through all array elements 
+  for i in range(n): # Traverse through all array elements
     for j in range(0, n-i-1): # Traverse the array from 0 to n-i-1 
         if arr[j] > arr[j+1]: 
             arr[j], arr[j+1] = arr[j+1], arr[j] # Swap if the element found is greater than the next element 
```

File: bubble_sort/Python/bubble_sort.py [hunk 3]
```diff
@@ -0,0 +1,14 @@
+def bubbleSort(arr): #Function that performs the bubbleSort
+
+  n = len(arr) #Finding the length of the array using the len function 
+
+  for i in range(n): # Traverse through all array elements 
+    for j in range(0, n-i-1): # Traverse the array from 0 to n-i-1
```
Comment (sangamcse) [inline bubble_sort/Python/bubble_sort.py]: Line contains following spacing inconsistencies:
- Trailing whitespaces.

*Origin: SpaceConsistencyBear, Section: `all.pyjava`.*

The issue can be fixed by applying the following patch:

```diff
--- a/tmp/tmpann5j0jr/bubble_sort/Python/bubble_sort.py
+++ b/tmp/tmpann5j0jr/bubble_sort/Python/bubble_sort.py
@@ -3,7 +3,7 @@
   n = len(arr) #Finding the length of the array using the len function 
 
   for i in range(n): # Traverse through all array elements 
-    for j in range(0, n-i-1): # Traverse the array from 0 to n-i-1 
+    for j in range(0, n-i-1): # Traverse the array from 0 to n-i-1
         if arr[j] > arr[j+1]: 
             arr[j], arr[j+1] = arr[j+1], arr[j] # Swap if the element found is greater than the next element 
               
```

File: bubble_sort/Python/bubble_sort.py [hunk 4]
```diff
@@ -0,0 +1,14 @@
+def bubbleSort(arr): #Function that performs the bubbleSort
+
+  n = len(arr) #Finding the length of the array using the len function 
+
+  for i in range(n): # Traverse through all array elements 
+    for j in range(0, n-i-1): # Traverse the array from 0 to n-i-1 
+        if arr[j] > arr[j+1]:
```
Comment (sangamcse) [inline bubble_sort/Python/bubble_sort.py]: Line contains following spacing inconsistencies:
- Trailing whitespaces.

*Origin: SpaceConsistencyBear, Section: `all.pyjava`.*

The issue can be fixed by applying the following patch:

```diff
--- a/tmp/tmpann5j0jr/bubble_sort/Python/bubble_sort.py
+++ b/tmp/tmpann5j0jr/bubble_sort/Python/bubble_sort.py
@@ -4,7 +4,7 @@
 
   for i in range(n): # Traverse through all array elements 
     for j in range(0, n-i-1): # Traverse the array from 0 to n-i-1 
-        if arr[j] > arr[j+1]: 
+        if arr[j] > arr[j+1]:
             arr[j], arr[j+1] = arr[j+1], arr[j] # Swap if the element found is greater than the next element 
               
 arr = [12, 96, 24, 48, 36, 60, 84, 72] # Declaring a sample array
```

File: bubble_sort/Python/bubble_sort.py [hunk 5]
```diff
@@ -0,0 +1,14 @@
+def bubbleSort(arr): #Function that performs the bubbleSort
+
+  n = len(arr) #Finding the length of the array using the len function 
+
+  for i in range(n): # Traverse through all array elements 
+    for j in range(0, n-i-1): # Traverse the array from 0 to n-i-1 
+        if arr[j] > arr[j+1]: 
+            arr[j], arr[j+1] = arr[j+1], arr[j] # Swap if the element found is greater than the next element
```
Comment (sangamcse) [inline bubble_sort/Python/bubble_sort.py]: Line contains following spacing inconsistencies:
- Trailing whitespaces.

*Origin: SpaceConsistencyBear, Section: `all.pyjava`.*

The issue can be fixed by applying the following patch:

```diff
--- a/tmp/tmpann5j0jr/bubble_sort/Python/bubble_sort.py
+++ b/tmp/tmpann5j0jr/bubble_sort/Python/bubble_sort.py
@@ -5,7 +5,7 @@
   for i in range(n): # Traverse through all array elements 
     for j in range(0, n-i-1): # Traverse the array from 0 to n-i-1 
         if arr[j] > arr[j+1]: 
-            arr[j], arr[j+1] = arr[j+1], arr[j] # Swap if the element found is greater than the next element 
+            arr[j], arr[j+1] = arr[j+1], arr[j] # Swap if the element found is greater than the next element
               
 arr = [12, 96, 24, 48, 36, 60, 84, 72] # Declaring a sample array
 bubbleSort(arr) # Calling the bubbleSort function and passing the array 
```

File: bubble_sort/Python/bubble_sort.py [hunk 6]
```diff
@@ -0,0 +1,14 @@
+def bubbleSort(arr): #Function that performs the bubbleSort
+
+  n = len(arr) #Finding the length of the array using the len function 
+
+  for i in range(n): # Traverse through all array elements 
+    for j in range(0, n-i-1): # Traverse the array from 0 to n-i-1 
+        if arr[j] > arr[j+1]: 
+            arr[j], arr[j+1] = arr[j+1], arr[j] # Swap if the element found is greater than the next element 
+
```
Comment (sangamcse) [inline bubble_sort/Python/bubble_sort.py]: Line contains following spacing inconsistencies:
- Trailing whitespaces.

*Origin: SpaceConsistencyBear, Section: `all.pyjava`.*

The issue can be fixed by applying the following patch:

```diff
--- a/tmp/tmpann5j0jr/bubble_sort/Python/bubble_sort.py
+++ b/tmp/tmpann5j0jr/bubble_sort/Python/bubble_sort.py
@@ -6,7 +6,7 @@
     for j in range(0, n-i-1): # Traverse the array from 0 to n-i-1 
         if arr[j] > arr[j+1]: 
             arr[j], arr[j+1] = arr[j+1], arr[j] # Swap if the element found is greater than the next element 
-              
+
 arr = [12, 96, 24, 48, 36, 60, 84, 72] # Declaring a sample array
 bubbleSort(arr) # Calling the bubbleSort function and passing the array 
 print ("Sorted array is:") # Printing the array after sorting it.
```

File: bubble_sort/Python/bubble_sort.py [hunk 7]
```diff
@@ -0,0 +1,14 @@
...
+  n = len(arr) #Finding the length of the array using the len function 
+
+  for i in range(n): # Traverse through all array elements 
+    for j in range(0, n-i-1): # Traverse the array from 0 to n-i-1 
+        if arr[j] > arr[j+1]: 
+            arr[j], arr[j+1] = arr[j+1], arr[j] # Swap if the element found is greater than the next element 
+              
+arr = [12, 96, 24, 48, 36, 60, 84, 72] # Declaring a sample array
+bubbleSort(arr) # Calling the bubbleSort function and passing the array
```
Comment (sangamcse) [inline bubble_sort/Python/bubble_sort.py]: Line contains following spacing inconsistencies:
- Trailing whitespaces.

*Origin: SpaceConsistencyBear, Section: `all.pyjava`.*

The issue can be fixed by applying the following patch:

```diff
--- a/tmp/tmpann5j0jr/bubble_sort/Python/bubble_sort.py
+++ b/tmp/tmpann5j0jr/bubble_sort/Python/bubble_sort.py
@@ -8,7 +8,7 @@
             arr[j], arr[j+1] = arr[j+1], arr[j] # Swap if the element found is greater than the next element 
               
 arr = [12, 96, 24, 48, 36, 60, 84, 72] # Declaring a sample array
-bubbleSort(arr) # Calling the bubbleSort function and passing the array 
+bubbleSort(arr) # Calling the bubbleSort function and passing the array
 print ("Sorted array is:") # Printing the array after sorting it.
 for i in range(len(arr)): 
   print(arr[i])
```

File: bubble_sort/Python/bubble_sort.py [hunk 8]
```diff
@@ -0,0 +1,14 @@
...
+
+  for i in range(n): # Traverse through all array elements 
+    for j in range(0, n-i-1): # Traverse the array from 0 to n-i-1 
+        if arr[j] > arr[j+1]: 
+            arr[j], arr[j+1] = arr[j+1], arr[j] # Swap if the element found is greater than the next element 
+              
+arr = [12, 96, 24, 48, 36, 60, 84, 72] # Declaring a sample array
+bubbleSort(arr) # Calling the bubbleSort function and passing the array 
+print ("Sorted array is:") # Printing the array after sorting it.
+for i in range(len(arr)):
```
Comment (sangamcse) [inline bubble_sort/Python/bubble_sort.py]: Line contains following spacing inconsistencies:
- Trailing whitespaces.

*Origin: SpaceConsistencyBear, Section: `all.pyjava`.*

The issue can be fixed by applying the following patch:

```diff
--- a/tmp/tmpann5j0jr/bubble_sort/Python/bubble_sort.py
+++ b/tmp/tmpann5j0jr/bubble_sort/Python/bubble_sort.py
@@ -10,5 +10,5 @@
 arr = [12, 96, 24, 48, 36, 60, 84, 72] # Declaring a sample array
 bubbleSort(arr) # Calling the bubbleSort function and passing the array 
 print ("Sorted array is:") # Printing the array after sorting it.
-for i in range(len(arr)): 
+for i in range(len(arr)):
   print(arr[i])
```

File: bubble_sort/Python/bubble_sort.py [hunk 9]
```diff
@@ -0,0 +1,14 @@
+def bubbleSort(arr):  # Function that performs the bubbleSort
+
+    n = len(arr)  # Finding the length of the array using the len function
+
+    for i in range(n):  # Traverse through all array elements
+      for j in range(0, n-i-1):  # Traverse the array from 0 to n-i-1
+          if arr[j] > arr[j+1]:  # Swap if the element found
+              arr[j], arr[j+1] = arr[j+1], arr[j]  # is greater than the next element
+
```
Comment (sangamcse) [inline bubble_sort/Python/bubble_sort.py]: Line contains following spacing inconsistencies:
- Trailing whitespaces.

*Origin: SpaceConsistencyBear, Section: `all.pyjava`.*

The issue can be fixed by applying the following patch:

```diff
--- a/tmp/tmpcxtxm6s5/bubble_sort/Python/bubble_sort.py
+++ b/tmp/tmpcxtxm6s5/bubble_sort/Python/bubble_sort.py
@@ -6,7 +6,7 @@
       for j in range(0, n-i-1):  # Traverse the array from 0 to n-i-1
           if arr[j] > arr[j+1]:  # Swap if the element found
               arr[j], arr[j+1] = arr[j+1], arr[j]  # is greater than the next element
-              
+
 arr = [12, 96, 24, 48, 36, 60, 84, 72] # Declaring a sample array
 bubbleSort(arr) # Calling the bubbleSort function and passing the array
 print ("Sorted array is:") # Printing the array after sorting it
```

File: bubble_sort/Python/bubble_sort.py [hunk 10]
```diff
@@ -0,0 +1,13 @@
+def bubbleSort(arr):  # Function that performs the bubbleSort
+
+    n = len(arr)  # Finding the length of the array using the len function
+
+    for i in range(n):  # Traverse through all array elements
+        for j in range(0, n-i-1):  # Traverse the array from 0 to n-i-1
+            if arr[j] > arr[j+1]:  # Swap if the element found
+                arr[j], arr[j+1] = arr[j+1], arr[j]  # is greater than next element
```
Comment (sangamcse) [inline bubble_sort/Python/bubble_sort.py]: The code does not comply to PEP8.

*Origin: PEP8Bear, Section: `all.autopep8`.*

The issue can be fixed by applying the following patch:

```diff
--- a/tmp/tmphbip7dpq/bubble_sort/Python/bubble_sort.py
+++ b/tmp/tmphbip7dpq/bubble_sort/Python/bubble_sort.py
@@ -5,8 +5,11 @@
     for i in range(n):  # Traverse through all array elements
         for j in range(0, n-i-1):  # Traverse the array from 0 to n-i-1
             if arr[j] > arr[j+1]:  # Swap if the element found
-                arr[j], arr[j+1] = arr[j+1], arr[j]  # is greater than next element
-arr = [12, 96, 24, 48, 36, 60, 84, 72] # Declaring a sample array
+                # is greater than next element
+                arr[j], arr[j+1] = arr[j+1], arr[j]
+
+
+arr = [12, 96, 24, 48, 36, 60, 84, 72]  # Declaring a sample array
 bubbleSort(arr)  # Calling the bubbleSort function and passing the array
 print("Sorted array is:")  # Printing the array after sorting it
 for i in range(len(arr)):
```

Comment (sangamcse) [inline bubble_sort/Python/bubble_sort.py]: E501 line too long (83 > 80 characters)

*Origin: PycodestyleBear (E501), Section: `all.autopep8`.*

File: bubble_sort/Python/bubble_sort.py [hunk 11]
```diff
@@ -0,0 +1,13 @@
...
+arr = [12, 96, 24, 48, 36, 60, 84, 72] # Declaring a sample array
```
Comment (sangamcse) [inline bubble_sort/Python/bubble_sort.py]: E305 expected 2 blank lines after class or function definition, found 0

*Origin: PycodestyleBear (E305), Section: `all.autopep8`.*

Comment (sangamcse) [inline bubble_sort/Python/bubble_sort.py]: E261 at least two spaces before inline comment

*Origin: PycodestyleBear (E261), Section: `all.autopep8`.*

File: bubble_sort/Python/bubble_sort.py (hunk 10 above)
Comment (sangamcse) [inline bubble_sort/Python/bubble_sort.py]: The code does not comply to PEP8.

*Origin: PEP8Bear, Section: `all.autopep8`.*

The issue can be fixed by applying the following patch:

```diff
--- a/tmp/tmpqk_1d0n3/bubble_sort/Python/bubble_sort.py
+++ b/tmp/tmpqk_1d0n3/bubble_sort/Python/bubble_sort.py
@@ -5,7 +5,10 @@
     for i in range(n):  # Traverse through all array elements
         for j in range(0, n-i-1):  # Traverse the array from 0 to n-i-1
             if arr[j] > arr[j+1]:  # Swap if the element found
-                arr[j], arr[j+1] = arr[j+1], arr[j]  # is greater than next element
+                # is greater than next element
+                arr[j], arr[j+1] = arr[j+1], arr[j]
+
+
 arr = [12, 96, 24, 48, 36, 60, 84, 72]  # Declaring a sample array
 bubbleSort(arr)  # Calling the bubbleSort function and passing the array
 print("Sorted array is:")  # Printing the array after sorting it
```

Comment (sangamcse) [inline bubble_sort/Python/bubble_sort.py]: E501 line too long (83 > 80 characters)

*Origin: PycodestyleBear (E501), Section: `all.autopep8`.*

File: bubble_sort/Python/bubble_sort.py [hunk 12]
```diff
@@ -0,0 +1,13 @@
+def bubbleSort(arr):  # Function that performs the bubbleSort
+
+    n = len(arr)  # Finding the length of the array using the len function
+
+    for i in range(n):  # Traverse through all array elements
+        for j in range(0, n-i-1):  # Traverse the array from 0 to n-i-1
+            if arr[j] > arr[j+1]:  # Swap if the element found
+                arr[j], arr[j+1] = arr[j+1], arr[j]  # is greater than next element
+arr = [12, 96, 24, 48, 36, 60, 84, 72]  # Declaring a sample array
```
Comment (sangamcse) [inline bubble_sort/Python/bubble_sort.py]: E305 expected 2 blank lines after class or function definition, found 0

*Origin: PycodestyleBear (E305), Section: `all.autopep8`.*

File: bubble_sort/Python/bubble_sort.py [hunk 13]
```diff
@@ -0,0 +1,13 @@
+def bubbleSort(arr):  # Function that performs the bubbleSort
+
+    n = len(arr)  # Finding the length of the array using the len function
+
+    for i in range(n):  # Traverse through all array elements
+        for j in range(0, n-i-1):  # Traverse the array from 0 to n-i-1
+            if arr[j] > arr[j+1]:  # Swap if the element found
+                arr[j], arr[j+1] = arr[j+1], arr[j]  # greater than next element
```
Comment (sangamcse) [inline bubble_sort/Python/bubble_sort.py]: The code does not comply to PEP8.

*Origin: PEP8Bear, Section: `all.autopep8`.*

The issue can be fixed by applying the following patch:

```diff
--- a/tmp/tmpjsnyunsd/bubble_sort/Python/bubble_sort.py
+++ b/tmp/tmpjsnyunsd/bubble_sort/Python/bubble_sort.py
@@ -6,6 +6,8 @@
         for j in range(0, n-i-1):  # Traverse the array from 0 to n-i-1
             if arr[j] > arr[j+1]:  # Swap if the element found
                 arr[j], arr[j+1] = arr[j+1], arr[j]  # greater than next element
+
+
 arr = [12, 96, 24, 48, 36, 60, 84, 72]    # Declaring a sample array
 bubbleSort(arr)  # Calling the bubbleSort function and passing the array
 print("Sorted array is:")  # Printing the array after sorting it
```

File: bubble_sort/Python/bubble_sort.py [hunk 14]
```diff
@@ -0,0 +1,13 @@
...
+arr = [12, 96, 24, 48, 36, 60, 84, 72]    # Declaring a sample array
```
Comment (sangamcse) [inline bubble_sort/Python/bubble_sort.py]: E305 expected 2 blank lines after class or function definition, found 0

*Origin: PycodestyleBear (E305), Section: `all.autopep8`.*

File: README.md [hunk 15]
```diff
@@ -19,7 +19,7 @@ This repository contains examples of various algorithms written on different pro
...
 | [Radix Sort](https://en.wikipedia.org/wiki/Radix_sort)                                          |                                       | [:octocat:](radix_sort/Cpp)                                       |                                       | [:octocat:](radix_sort/Python)        |
 | [Binary Search](https://en.wikipedia.org/wiki/Binary_search_algorithm)                          |                                       | [:octocat:](binary_search/Cpp)        |                                       | [:octocat:](binary_search/Python)     |
-| [Bubble Sort](https://en.wikipedia.org/wiki/Bubble_sort)                                        | [:octocat:](bubble_sort/C)            | [:octocat:](bubble_sort/Cpp)          | [:octocat:](bubble_sort/Java)         |                                       |
+| [Bubble Sort](https://en.wikipedia.org/wiki/Bubble_sort)                                        | [:octocat:](bubble_sort/C)            | [:octocat:](bubble_sort/Cpp)          | [:octocat:](bubble_sort/Java)| [:octocat:](bubble_sort/Python)|
```
Comment (sangamcse) [inline README.md]: Don't remove spaces of Java column

File: bubble_sort/Python/bubble_sort.py [hunk 16]
```diff
@@ -0,0 +1,15 @@
+def bubbleSort(arr):  # Function that performs the bubbleSort
```
Comment (sangamcse) [inline bubble_sort/Python/bubble_sort.py]: the function name is not as per documentation.

**Comment by sangamcse** [comment]
> The Travis CI build is failing in Python. I'm not sure why this is happening. Once, the Travis build passes, I'll update the PR with the guidelines and let you know, so, that you can merge the PR.

@sameerchoubey the Travis checks all things including your PR. If you are thinking that it will only check your code then you are wrong.

File: README.md [hunk 17]
```diff
@@ -19,7 +20,7 @@ This repository contains examples of various algorithms written on different pro
...
 | [Radix Sort](https://en.wikipedia.org/wiki/Radix_sort)                                          |                                       | [:octocat:](radix_sort/Cpp)                                       |                                       | [:octocat:](radix_sort/Python)        |
 | [Binary Search](https://en.wikipedia.org/wiki/Binary_search_algorithm)                          |                                       | [:octocat:](binary_search/Cpp)        |                                       | [:octocat:](binary_search/Python)     |
-| [Bubble Sort](https://en.wikipedia.org/wiki/Bubble_sort)                                        | [:octocat:](bubble_sort/C)            | [:octocat:](bubble_sort/Cpp)          | [:octocat:](bubble_sort/Java)         |                                       |
+| [Bubble Sort](https://en.wikipedia.org/wiki/Bubble_sort)                                        | [:octocat:](bubble_sort/C)            | [:octocat:](bubble_sort/Cpp)          | [:octocat:](bubble_sort/Java)         | [:octocat:](bubble_sort/Python) |
```
Comment (sangamcse) [inline README.md]: Give six blank space more in the python column.

File: README.md [hunk 18]
```diff
@@ -71,3 +72,4 @@ Feel free to contact us at our [Gitter channel](https://gitter.im/NITSkmOS/algo)
 
 Only project maintainers should merge a PR.
 Other members can add their reviews to a PR but the merging should be done by only a project maintainer.
+
```
Comment (sangamcse) [inline README.md]: Redundant change

File: README.md [hunk 19]
```diff
@@ -4,6 +4,7 @@
 [![NITSkmOS Algorithms](https://img.shields.io/badge/NITSkmOS-Algorithms-blue.svg)](https://github.com/NITSkmOS/Algorithms)
 [![Build Status](https://travis-ci.com/NITSkmOS/Algorithms.svg?branch=master)](https://travis-ci.com/NITSkmOS/Algorithms)
 [![GitHub](https://img.shields.io/github/license/mashape/apistatus.svg)](https://github.com/NITSkmOS/Algorithms/blob/master/LICENSE)
+[![OpenHub](https://www.openhub.net/p/NITSkmOS-algo/widgets/project_thin_badge?format=gif)](https://www.openhub.net/p/NITSkmOS-algo?ref=Thin+badge)
```
Comment (sangamcse) [inline README.md]: Rebase your PR 😉 
It will then go from this PR automatically.

File: bubble_sort/Python/bubble_sort.py [hunk 20]
```diff
@@ -0,0 +1,20 @@
+def bubble_sort(arr):  # Function that performs the bubbleSort
+
```
Comment (sangamcse) [inline bubble_sort/Python/bubble_sort.py]: Get the idea about the comments from other python code in this repo.

File: bubble_sort/Python/bubble_sort.py [hunk 21]
```diff
@@ -0,0 +1,23 @@
+def bubble_sort(arr):  # Function that performs the bubbleSort
+    """
+    The bubble_sort funtion takes an array as an argument
+    and then sorts the elements present in that array.
+    """
```
Comment (sangamcse) [inline bubble_sort/Python/bubble_sort.py]: Add `:param arr:` and its brief description also

File: bubble_sort/Python/bubble_sort.py [hunk 22]
```diff
@@ -0,0 +1,23 @@
...
+    for i in range(n):  # Traverse through all array elements
+        for j in range(0, n-i-1):  # Traverse the array from 0 to n-i-1
+            if arr[j] > arr[j+1]:  # Swap if the element found
+                arr[j], arr[j+1] = arr[j+1], arr[j]  # greater than next element
+
+
+def main():
+    arr = [12, 96, 24, 48, 36, 60, 84, 72]    # Declaring a sample array
+    bubble_sort(arr)  # Calling the bubbleSort function and passing the array
+    print("Sorted array is:")  # Printing the array after sorting it
```
Comment (sangamcse) [inline bubble_sort/Python/bubble_sort.py]: No need to create a for loop here. Use `format` and `join` function

File: README.md [hunk 23]
```diff
@@ -19,7 +20,7 @@ This repository contains examples of various algorithms written on different pro
...
 | [Radix Sort](https://en.wikipedia.org/wiki/Radix_sort)                                          |                                       | [:octocat:](radix_sort/Cpp)                                       |                                       | [:octocat:](radix_sort/Python)        |
 | [Binary Search](https://en.wikipedia.org/wiki/Binary_search_algorithm)                          |                                       | [:octocat:](binary_search/Cpp)        |                                       | [:octocat:](binary_search/Python)     |
-| [Bubble Sort](https://en.wikipedia.org/wiki/Bubble_sort)                                        | [:octocat:](bubble_sort/C)            | [:octocat:](bubble_sort/Cpp)          | [:octocat:](bubble_sort/Java)         |                                       |
+| [Bubble Sort](https://en.wikipedia.org/wiki/Bubble_sort)                                        | [:octocat:](bubble_sort/C)            | [:octocat:](bubble_sort/Cpp)          | [:octocat:](bubble_sort/Java)         |       [:octocat:](bubble_sort/Python) |
```
[the remark this answers is not in the document]
Comment (sangamcse) [reply README.md]: I didn't say line, I said blank space(white space). Also, it should be after `[:octocat:](bubble_sort/Python)` and not before that. See other lines of README no

**Comment by sameerchoubey** [comment]
@sangamcse All done. Please review.

**Comment by sangamcse** [comment]
ack 9cc4833

**Comment by sangamcse** [comment]
@sangamcse ff

**Comment by sangamcse** [comment]
Hey! I'm [GitMate.io](https://gitmate.io)! This pull request is being fastforwarded automatically. Please **DO NOT** push while fastforward is in progress or your changes would be lost permanently :warning:

**Comment by sangamcse** [comment]
Automated fastforward with [GitMate.io](https://gitmate.io) was successful! :tada:

**Comment by sangamcse** [comment]
ack 9cc48337416fc7854665a481711aa7b4d0c3ed52

**Pull request merged**